src/libDirectoryService/PoW1Processing.cpp

```diff
@@ -211,7 +212,18 @@ bool DirectoryService::ParseMessageAndVerifyPOW1(
                 return false;
             }
 
-            m_allPoW1s.push_back(make_pair(key, nonce));
+            if (TEST_NET_MODE
```

Comment:
Don't you think we should check if the DS belongs to whitelist before verifying the PoW submission by it ?